extensions/interactions/base.py

```diff
@@ -143,6 +143,25 @@ def _stats_log_template(self):
             return '{{answer}}'
 
     @property
+    def rules(self):
+        if self._cached_rules is not None:
+            return self._cached_rules
+
+        rules_index_dict = json.loads(
```

Comment:
Probably, but there is also a TODO to split up the json file into individual interactions, so I think worrying about caching after that is probably best.